core/vm/vm.go

```diff
@@ -201,6 +199,35 @@ func (self *Vm) Run(contract *Contract, input []byte) (ret []byte, err error) {
 
 						continue
 					}
+				case CREATE:
```

Comment:
I see now. It's because of the error. Why not change the definition of the call to return an error instead.